features/Report/CustomReport/GroupLoanReport/index.tsx

```diff
@@ -66,7 +74,7 @@ export const GroupLoanReport = () => {
                   totalRow: ['Total Amount', '', '₦104,200.65', '₦104,200.65']
                 }}
                 columns={COLUMN}
-                data={groupLoanReportList}
+                data={downloadData}
```

Comment:
You shouldn't use the download data on the table here. This defeats the purpose of separating the report endpoint from the table endpoint

```suggestion
                data={groupLoanReportList}
```